.github/workflows/tests.yml

```diff
@@ -63,13 +88,87 @@ jobs:
             bin/
           key: wasm-module-build-${{ hashFiles('modsecurity-wasm-filter/**') }}
 
-      - name: "Spin up server and envoy"
+      - name: "Spin up local server and envoy"
         shell: bash
         run: |
           docker run --rm -d -p 8000:10080 jcchavezs/httpmole -response-status=200 &
```

Comment:
This should be rebased.

Reply:
Rebased, let's see if tests are going to be okay. 
I also removed ` docker run --rm -d -p 8000:10080 jcchavezs/httpmole -response-status=200 &` being useless now